### PR TITLE
prevent no testcases error message

### PR DESCRIPTION
**Pull request by bmcutler**

### CODE REVIEW

File: site/public/css/server.css [hunk 1]
```diff
@@ -107,9 +107,9 @@ input[type="checkbox"] {
  * TABLE STYLES
  ***************************/
 table {
-    width: 100%;
-    text-align: center;
-    border-collapse: collapse;
+    /*width: 100%;*/
+    /*text-align: center;*/
+    /*border-collapse: collapse;*/
```
Comment (MasterOdin) [inline site/public/css/server.css]: Why this change?

Comment (bmcutler) [inline site/public/css/server.css]: This global was causing the rainbow grades summary to go 100% width.

Comment (MasterOdin) [inline site/public/css/server.css]: Fair. I've moved this to a class definition since this change negatively affected students/graders tables.
